ansible/roles/protobuf_compiler/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+- name: Download protoc zip file
+  ansible.builtin.get_url:
+    url: "{{ protoc_zip_url }}"
+    dest: "/tmp/"
+    mode: '0644'
+
+- name: Ensure destination directory exists
+  ansible.builtin.file:
+    path: "{{ protoc_dest_path }}"
+    state: directory
+    mode: '0755'
+
+- name: Unzip protoc to the destination
+  ansible.builtin.unarchive:
+    src: "{{ protoc_zip_path }}/{{ protoc_zip_file_name }}"
+    dest: "{{ protoc_dest_path }}"
+    remote_src: true
+# TODO: Create an empty file with version and then check if it's exists
```

Comment:
Create a file with current version in name after installation of protoc. If file exists we can skip all these tasks